src/sentry/integrations/github/webhook.py

```diff
@@ -75,11 +85,21 @@ def __call__(self, event, host=None):
             external_id = event['installation']['id']
             if host:
                 external_id = u'{}:{}'.format(host, event['installation']['id'])
-            integration = Integration.objects.get(
-                external_id=external_id,
-                provider=self.provider,
-            )
-            self._handle_delete(event, integration)
+            try:
+                integration = Integration.objects.get(
+                    external_id=external_id,
+                    provider=self.provider,
+                )
+            except Integration.DoesNotExist:
+                logger.info(
+                    'github.deletion-missing-integration',
+                    extra={
+                        'event_data': event,
+                        'external_id': external_id,
+                    }
+                )
+            else:
+                self._handle_delete(event, integration)
```

Comment:
You could put this in the `try` side. ⛳️

Reply:
why do we want this in the try block? it doesn't seem to me like `_handle_delete` can raise an `Integration.DoesNotExist` error